README.md

```diff
@@ -147,3 +147,75 @@ To access PhpMyAdmin, run `docker ps` to get its dynamic port.
 * Username: root
 * Password: root
 
+## Install and setup XDebug
```

Comment:
Can you just specify which file and part of the file should be uncommented? No need to repeat all the lines. It will be easier to read and to maintain for the README. You can see how it's done here for example: https://github.com/expresso-php/expresso-php/pull/67/files#diff-04c6e90faac2675aa89e2176d2eec7d8R149